tokio/src/sync/notify.rs

```diff
@@ -109,7 +109,11 @@ type WaitList = LinkedList<Waiter, <Waiter as linked_list::Link>::Target>;
 /// [`Semaphore`]: crate::sync::Semaphore
 #[derive(Debug)]
 pub struct Notify {
-    state: AtomicU8,
+    // this uses 2 bits to store one of `EMPTY`,
```

Comment:
very, very minor nit: other sentences in this comment start with capital letters; let's be consistent:
```suggestion
    // This uses 2 bits to store one of `EMPTY`,
```